python/lsst/cell_coadds/_stitched_coadd.py

```diff
@@ -175,3 +179,25 @@ def asExposure(self) -> ExposureF:
         result.setPsf(self.psf)
 
         return result
+
+    @property
+    def stitched_ap_corr_map(self) -> ApCorrMap:
```

Comment:
I should make this just `ap_corr_map` or something (i.e., drop the `stitched` prefix) and be consistent with `psf`.